python/datamodel/nodesim/datamodel.py

```diff
@@ -3,35 +3,13 @@
 
 @author: Arthur Valadares
 '''
+from __future__ import absolute_import
+
 from pcc.set import pcc_set
 from pcc.attributes import primarykey, dimension
 from pcc.projection import projection
-
-class Vector3(object):
-    X = 0
-    Y = 0
-    Z = 0
-
-    def __init__(self, X, Y, Z):
-        self.X = X
-        self.Y = Y
-        self.Z = Z
-
-    def __json__(self):
-        return self.__dict__
-
-    def __str__(self):
-        return self.__dict__.__str__()
-
-    def __eq__(self, other):
-        return (isinstance(other, Vector3) and (other.X == self.X and other.Y == self.Y and other.Z == self.Z))
-
-    def __ne__(self, other):
-        return not self.__eq__(other)
-
-    @staticmethod
-    def __decode__(dic):
-        return Vector3(dic['X'], dic['Y'], dic['Z'])
+import numpy as np
```

Comment:
I just realized that datamodel imports are expected to be present on both client and server, even if one of them do not need the imports. (unless you import within the function that you are only using in client or server). We need to give a definition file or something for remote "pip install <package you want>" or something.